lib/overcommit/hook/prepare_commit_msg/replace_branch.rb

```diff
@@ -17,21 +43,21 @@ def run
       Overcommit::Utils.log.debug("Writing #{commit_message_filename} with #{new_template}")
 
       modify_commit_message do |old_contents|
-        "#{new_template} #{old_contents}"
+        "#{new_template}#{old_contents}"
       end
 
       :pass
     end
 
     def new_template
-      @new_template ||= Overcommit::GitRepo.current_branch.gsub(branch_pattern, replacement_text)
+      @new_template ||= Overcommit::GitRepo.current_branch.gsub(branch_pattern, replacement_text).strip
```

Comment:
I believe RuboCop is complaining about this line, @jethrodaniel. Should be easy to address.